client/analytics/index.js

```diff
@@ -8,14 +8,27 @@ import { Component, Fragment } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import { Card, Search } from '@woocommerce/components';
 import Header from 'layout/header';
 
 export default class extends Component {
+	constructor( props ) {
+		super( props );
+		this.onChange = this.onChange.bind( this );
+	}
+
+	onChange() {
+		// Here we can do whatever we want with `results`
+		// console.log( results );
```

Comment:
This will be an array of whatever's returned from `getOptionCompletion`, so you can see that it's the product name & ID. In filters, we can use this to set a query param.